crypto/cryptlib.c

```diff
@@ -120,6 +124,18 @@ void OPENSSL_cpuid_setup(void)
 # endif
 
 # if defined(_WIN32_WINNT) && _WIN32_WINNT>=0x0333
+#ifdef OPENSSL_SYS_WIN_CORE
+TRACELOGGING_DEFINE_PROVIDER(
+    g_hProvider,
+    "OpenSSL",
+    /*{5ee02bf4-33cf-53cf-ddc9-f18679e677dd}*/
+    (0x5ee02bf4, 0x33cf, 0x53cf, 0xdd, 0xc9, 0xf1, 0x86, 0x79, 0xe6, 0x77, 0xdd));
+int OPENSSL_isservice(void)
+{
+    /*Core API cannot interact with GUI*/
```

Comment:
Styling nit. Even though styling guide is not explicit about it, I don't thing you'll find one-line comments without spaces after opening or prior closing asterisks.

Also, we intend pre-processor directives, but by adding spaces between hash-tag and directive itself.